### ISSUE TITLE
define the metric of success

### ISSUE THREAD

**Issue by chunhualiao** [asherliu/researchHOWTO#2]
How to evaluate if this tutorial is effective ?  Should you have some (incremental) tests to check if readers grasp the key concepts and skills ?  Or a reader shows he/she publishes a paper (judging by end results)?

**Comment by chunhualiao** [comment]
I just found this: 

https://en.m.wikipedia.org/wiki/Donald_Kirkpatrick

The four levels of Kirkpatrick's evaluation model are as follows:

Reaction - what participants thought and felt about the training (satisfaction; "smile sheets")

Learning - the resulting increase in knowledge and/or skills, and change in attitudes. This evaluation occurs during the training in the form of either a knowledge demonstration or test.

Behavior - transfer of knowledge, skills, and/or attitudes from classroom to the job (change in job behavior due to training program). This evaluation occurs 3–6 months post training while the trainee is performing the job. Evaluation usually occurs through observation.

Results - the final results that occurred because of attendance and participation in a training program (can be monetary, performance-based, etc.)

**Comment by hpda-lab** [comment]
When it goes to writing a paper, ``Think Like A Reader`` is the most important trick! Two ways to create such a "reader".

- You find a fresh eye (like your friend) to check whether your paper can be understood.
- After a context switch of you (like working on another project, after an intensive exercise, long vacation, or simply a good sleep), you yourself read this paper again and see if you could understand it. 

In short, the evaluation of success often comes from outsider instead of yourself. 

Probably, the end goal is to put together a tool. Once you feed in the paper, this tool can help you judge the extent of progresses.
